ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml

```
@@ -71,43 +71,67 @@ tests:
 - as: e2e-aws-ovn-techpreview
   steps:
     cluster_profile: aws
-    workflow: openshift-e2e-aws-capi
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    workflow: openshift-e2e-vsphere-capi
 - as: e2e-aws-capi-techpreview
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
     cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
     test:
     - as: test
       commands: make e2e
       from: src
       resources:
         requests:
           cpu: 100m
-    workflow: openshift-e2e-aws-capi
+    workflow: openshift-e2e-vsphere-capi
```

Needs to stay as the correct platform
```suggestion
    workflow: openshift-e2e-aws
```